snip-gui/src/app/Components/snippet-view/snippet-view.component.html

```diff
@@ -1,6 +1,31 @@
 <!-- This component is the page that a user will view a snippet on -->
 <h4>User Snippet</h4>
 
+
```

Comment:
What's this for? I made a bunch of changes to this file, so may be best to get rid of it